tools/check

```diff
@@ -241,6 +241,21 @@ should_run_build_runner() {
     return 1
 }
 
+run_l10n() {
+    local output_path=lib/generated/l10n
+
+    files_check "${output_path}"/'*'.dart \
```

Comment:
nit: pull the whole pathspec `lib/generated/l10n/'*'.dart` out as a local variable

That way there's a natural place for a comment clarifying that this string is meant as a Git pathspec. Otherwise it looks an awful lot like a shell glob (which it would be if the `*` weren't escaped with those single quotes), which would have somewhat different semantics. See `run_pigeon` for an example (though that example also makes it an array, which isn't needed here since there's just one pathspec).